### PR TITLE
Two functions: one for adding bond slave, one for getting veth peer index

### PR DESCRIPTION
**Pull request by jjastrze-ovh**
As in topic. Please review this.

### CODE REVIEW

File: ioctl_linux.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	// ETH_SS_FEATURES are device feature names
+	ETH_SS_FEATURES
+	// ETH_SS_RSS_HASH_FUNCS is RSS hush function names
+	ETH_SS_RSS_HASH_FUNCS
+)
+
+// IfreqSlave is a struct for ioctl bond manipulation syscalls.
+// It is used to assign slave to bond interface with Name.
+type IfreqSlave struct {
+	Name  [syscall.IFNAMSIZ]byte
```
Comment (aboch) [inline ioctl_linux.go]: Can you switch to the `unix` pkg instead of `syscall` ?

File: link_linux.go [hunk 2]
```diff
@@ -2256,3 +2256,57 @@ func parseGTPData(link Link, data []syscall.NetlinkRouteAttr) {
...
+		return fmt.Errorf("Failed to enslave %q to %q, errno=%v", link.Attrs().Name, master.Attrs().Name, errno)
+	}
+	return nil
+}
+
+// VethPeerIndex get veth peer index.
+func VethPeerIndex(link *Veth) (int, error) {
+	fd, err := getSocketUDP()
+	if err != nil {
+		return 0, err
```
Comment (aboch) [inline link_linux.go]: why not returning `-1` as in all other error cases below

**Comment by aboch** [comment]
Thanks for the contributions.

Couple of minor comments. Also, ideally this should be two commits: One for `VethPeerIndex` and its testing code, and one commit for `LinkSetBondSlave` and its testing code.

**Comment by jjastrze-ovh** [comment]
Thanks @aboch for review. I have fixed the two files you requested :)

**Comment by aboch** [comment]
Please do not add a new commit to take care of the review comments.
Please use `git rebase -i` and fix the target commit or just squash them into one before pushing the PR.

**Comment by jjastrze-ovh** [comment]
@aboch I have put everything into one commit.

**Comment by aboch** [comment]
Thanks @jjastrze-ovh / @phob0s-pl 

LGTM

**Pull request merged**